perceiver/model/image/optical_flow.py

```diff
@@ -30,58 +30,42 @@
 @dataclass
 class OpticalFlowEncoderConfig(EncoderConfig):
     image_shape: Tuple[int, int] = (368, 496)
-    num_temporal_channels: int = 2
-    num_image_input_channels: int = 27
-    num_image_output_channels: int = 64
+    patch_size: int = 3
+    num_patch_hidden_channels: int = 64
     num_frequency_bands: int = 64
-    spatial_downsample: int = 1
-    temporal_downsample: int = 2
 
 
 @dataclass
 class OpticalFlowDecoderConfig(DecoderConfig):
     output_image_shape: Tuple[int, int] = (368, 496)
-    num_output_image_channels: int = 2
-    num_output_query_channels: int = 322
     rescale_factor: float = 100.0
 
 
 class OpticalFlowInputAdapter(InputAdapter):
     def __init__(
         self,
         image_shape: Tuple[int, int],
-        num_image_input_channels: int,
-        num_image_output_channels: int,
-        num_temporal_channels: int,
+        patch_size: int,
+        num_patch_hidden_channels: int,
         num_frequency_bands: int,
-        spatial_downsample: int = 1,
-        temporal_downsample: int = 2,
+        num_temporal_channels: int = 2,
```

Comment:
What about removing `num_temporal_channels` at all (and hard-code its value below)?